src/Components/Aspire.NATS.Net/README.md

```diff
@@ -80,7 +80,7 @@ The .NET Aspire NATS component supports [Microsoft.Extensions.Configuration](htt
 Also you can pass the `Action<NatsClientSettings> configureSettings` delegate to set up some or all the options inline, for example to disable health checks from code:
 
 ```csharp
-    builder.AddNats("nats", settings => settings.HealthChecks = false);
+builder.AddNatsClient("nats", settings => settings.HealthChecks = false);
```

Comment:
I will create a PR to fix that in the other components. I have missed that this is for the client when I did the renaming (in READMEs).